### PR TITLE
[Fetch] Manage workspace in Fetch's PC by .rosinstall

### PR DESCRIPTION
**Pull request by 708yamaguchi**
In this Pull Request, I add `jsk_fetch.rosinstall` which can reproduce the workspace in fetch' PC.

This `jsk_fetch.rosinstall` is based on https://github.com/jsk-ros-pkg/jsk_robot/pull/1017, https://github.com/jsk-ros-pkg/jsk_robot/pull/1048, https://github.com/jsk-ros-pkg/jsk_robot/pull/1050. https://github.com/jsk-ros-pkg/jsk_robot/pull/1051, https://github.com/jsk-ros-pkg/jsk_robot/pull/1052, https://github.com/jsk-ros-pkg/jsk_robot/pull/1053, https://github.com/jsk-ros-pkg/jsk_robot/pull/1054, so please merge after these PR are merged.

Currently, workspace in fetch's PC is managed by this `jsk_fetch.rosinstall`

I will keep this setup for several weeks with @sktometometo.
If some problems occur, then I will re-make PR.

### CODE REVIEW

File: jsk_fetch_robot/jsk_fetch.rosinstall [hunk 1]
```diff
@@ -1,25 +1,67 @@
...
-    version: device
-# TODO(furushchev): describe why this is needed.
+    local-name: furushchev/image_pipeline
+    uri: https://github.com/furushchev/image_pipeline.git
+    version: develop
+# To send lifelog data to musca
+- git:
+    local-name: furushchev/mongodb_store
+    uri: https://github.com/furushchev/mongodb_store.git
+    version: develop
```
Comment (knorth55) [inline jsk_fetch_robot/jsk_fetch.rosinstall]: `mongodb_store` is now released.
PR2 now uses `0.4.4` and works fine.

Comment (708yamaguchi) [reply jsk_fetch_robot/jsk_fetch.rosinstall]: Thank you for your information.

I pushed commit.

**Comment by knorth55** [comment]
This PR depends on https://github.com/jsk-ros-pkg/jsk_robot/pull/1062.

**Comment by knorth55** [comment]
merge after https://github.com/jsk-ros-pkg/jsk_robot/pull/1062.

**Comment by k-okada** [comment]
failing on travis
```
                                                                                
FAILURE:



                                                                                
[/home/travis/ros/ws_jsk_robot/src/jsk_robot/jsk_fetch_robot/jsk_fetch_startup/launch/fetch_gazebo_bringup.launch]:



                                                                                
	roslaunch file contains multiple nodes named [/sound_play].



                                                                                
Please check all <node> 'name' attributes to make sure they are unique.



                                                                                
Also check that $(anon id) use different ids.



                                                  
make[3]: Entering directory `/home/travis/ros/ws_jsk_robot/build/jsk_fetch_startup'



                                                                                
catkin_generated/env_cached.sh /usr/bin/python /opt/ros/indigo/share/catkin/cmake/test/run_tests.py /home/travis/ros/ws_jsk_robot/build/jsk_fetch_startup/test_results/jsk_fetch_startup/roslaunch-check_launch_fetch_gazebo_bringup.launch.xml /usr/bin/cmake\ -E\ make_directory\ /home/travis/ros/ws_jsk_robot/build/jsk_fetch_startup/test_results/jsk_fetch_startup /home/travis/ros/ws_jsk_robot/src/jsk_robot/jsk_fetch_robot/jsk_fetch_startup/scripts/roslaunch-check\ -o\ '/home/travis/ros/ws_jsk_robot/build/jsk_fetch_startup/test_results/jsk_fetch_startup/roslaunch-check_launch_fetch_gazebo_bringup.launch.xml'\ '/home/travis/ros/ws_jsk_robot/src/jsk_robot/jsk_fetch_robot/jsk_fetch_startup/launch/fetch_gazebo_bringup.launch'\ 
```

https://api.travis-ci.org/v3/job/524957777/log.txt

**Comment by knorth55** [comment]
Waiting for these PRs:
- [ ] #1017
- [ ] #1048 
- [ ] #1049
- [ ] #1050 
- [ ] #1051 
- [ ] #1052 
- [ ] #1053 
- [ ] #1054
- [ ] #1062

**Comment by k-okada** [comment]
@knorth55 @708yamaguchi please resolve conflicts

**Comment by knorth55** [comment]
all other PRs are merged, so I squashed all commits and make this PR simple.
cc. @708yamaguchi

**Comment by 708yamaguchi** [comment]
@knorth55 
Thank you very much!

**Pull request merged**